src/mgr/BaseMgrStandbyModule.cc

```diff
@@ -65,7 +65,7 @@ ceph_get_module_option(BaseMgrStandbyModule *self, PyObject *args)
 {
   char *what = nullptr;
   char *prefix = nullptr;
-  if (!PyArg_ParseTuple(args, "ss:ceph_get_module_option", &what, &prefix)) {
+  if (!PyArg_ParseTuple(args, "s|z:ceph_get_module_option", &what, &prefix)) {
```

Comment:
nit: I think if you use ``z`` you dont' need ``|``.. see for instance ``ceph_store_set``

Reply:
Fixed